core/src/main/java/com/decibeltx/studytracker/core/service/impl/StatisticsServiceImpl.java

```diff
@@ -39,6 +43,7 @@ public Statistics getCurrent() {
     statistics.setUserCount(userService.count());
     statistics.setActivityCount(activityService.count());
     statistics.setActiveUserCount(userService.countActiveUsers());
+    statistics.setEntryTemplateCount(entryTemplateService.count());
```

Comment:
Again, if we are not going to display this on the main page, let's not add the extra query.